src/pages/ProfilePage.js

```diff
@@ -0,0 +1,16 @@
+// import MissionFilter from '../components/Profile/MissionFilter';
```

Comment:
- [ ] **Required:** Kindly remove all unused codes from your codebase. It makes code clean, more readable and professional.